step-templates/mysql-backup-database.json

```diff
@@ -0,0 +1,105 @@
+      "Octopus.Action.EnabledFeatures": ""
+    },
+    "Parameters": [
+      {
+        "Id": "7030ad54-4e10-4854-998a-ca91e3b490a3",
+        "Name": "createMySQLServerName",
+        "Label": "Server",
+        "HelpText": "Name or IP of the MySQL server",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "07dbf251-a982-4cc5-b6c1-dc0af50c519c",
+        "Name": "createusername",
+        "Label": "Username",
+        "HelpText": "\tUsername with rights to dump database",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "ac42a150-b7e7-4414-9dec-f18e105eaedf",
+        "Name": "createUserPassword",
+        "Label": "Password",
+        "HelpText": "Password for the user account",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "62ffb949-c5db-4151-bb29-47a37586bda5",
+        "Name": "createDatabaseName",
```

Comment:
Can you namespace this parameter, e.g. `BackupMySQL.DatabaseName` (the term `create` feels wrong here) or something similar?